### DISCUSSION TITLE
Include Branch and Commit names in Release Notes

### DISCUSSION THREAD

**Discussion by vovaklh** [codemagic-ci-cd/codemagic-docs#1244] [Q&A] [answered]
Hi! Can anyone tell how to add branch name and commit name in release notes when distributing to Firebase and Testflight?

**Comment by K-Codemagic** [comment, marked as answer]
You can create a script to add details in the release_notes. Something like this:

```
CURRENT_BRANCH=$(git branch --show-current)
GIT_COMMIT_MESSAGE=$(git log --format=%B -n 1 $CM_COMMIT)
```

and then add that to the release_notes.json using cat or sed. Example:

```
RELEASENOTES="${CURRENT_BRANCH}${GIT_COMMIT_MESSAGE}"
chmod +x $CM_BUILD_DIR/release_notes.json
cat <<< "$RELEASENOTES" > $CM_BUILD_DIR/release_notes.json
```

**Reply by vovaklh** [reply]
@K-Codemagic 
Thank you! But we should use release_notes.txt insted of release_notes.json to make script working

**Reply by K-Codemagic** [reply]
You'll need release_notes.json for Testflight/App store.
